src/asm/rgbasm.5

```diff
@@ -1036,10 +1055,9 @@ delim $$
 delim off
```

Comment:
@bentley, do you know why this is here?

Reply:
This prevents later use of $ in the document from unintentionally creating inline equations.